dunwoody_disaster/CharacterFactory.py

```diff
@@ -10,12 +10,14 @@ def __init__(self):
         self.maxHealth = 0
         self.curMagic = 0
         self.maxMagic = 0
-        self.curMechanical = 0
-        self.maxMechanical = 0
+        self.curStamina = 0
+        self.maxStamina = 0
 
-        # What are these? --Cooper
-        self.defense = 0
-        self.magicDefense = 0
+        self.meters = {
+            "health": self.maxHealth,
+            "magic": self.maxMagic,
+            "stamina": self.maxStamina,
+        }
```

Comment:
What is the point of this? I have something similar but a little different on my latest branch. Don't want to cause any conflicts.